lib/src/service/scroll_service.dart

```diff
@@ -27,6 +27,10 @@ abstract class AppFlowyScrollService implements AutoScrollerService {
   /// scroll controller
   ScrollController get scrollController;
 
+  bool get implecet;
```

Comment:
```suggestion
  bool get implicit;
```
By the name of the variable, I am unsure of its purpose. Maybe we can name it something better?